public/index.css

```diff
@@ -0,0 +1,66 @@
+:root {
```

Comment:
Some of this is copied CSS from Josue's component CSS files from our react project